### ISSUE TITLE
change course direction

### ISSUE THREAD

**Issue by MichaelVetter1979** [Courseplay/Courseplay_FS22#2635]
**What is the problem this feature/enhancement solves?**
Mowing a field using the Pöttinger for example is done starting on the outside. Baling is done better from inside out.

**Describe the solution you'd like**
Add a possibility to change the direction (Start/Stop) of a saved course.

**Comment by pvaiko** [comment]
Not sure what course generator settings you use, but you can try generating two courses, with the same setting, one starting on headland, the other in the middle. The generator internally always generates from the outside and just reverses the course when you set starting from the middle.

**Comment by MichaelVetter1979** [comment]
> Not sure what course generator settings you use, but you can try generating two courses, with all the same settings, except one starting on headland, the other in the middle. The generator internally always generates from the outside and just reverses the course when you set starting from the middle.

Yes, I am aware of that. However, I have often forgotten to generate the second course together with the first. And if this is not done with the same field positions, the courses are not congruent. A possibility to let a course travel from the end to the beginning would certainly be a nice addition there.
